.flake8

```diff
@@ -1,4 +1,65 @@
 [flake8]
-# For error codes, see this https://flake8.pycqa.org/en/latest/user/error-codes.html
-ignore = F401, F404, F406, F541, F841,
 max-line-length = 100
+statistics = true
+noqa-require-code = true
+exclude =
+    CVS
+    .bzr
+    .hg
+    .git
+    __pycache__
+    .tox
+    .nox
+    .eggs
+    *.egg
+    .venv
+    venv
+    .pytest_cache
+    .mypy_cache
+    .nox
+    build
+    test_bot
+
+# For error codes, see this https://flake8.pycqa.org/en/latest/user/error-codes.html
+ignore =
+    # not compatiable with black
+    E203,W503,
+    # ignore linebreak after `/`
+    W504,
+    # line too long. Acceptable at the moment since these are mostly comments
+    E501,
+    # assigning lambdas to variables
+    E731,
+    # ambiguous variable names
+    E741,
+    # may be undefined, or defined from star imports
+    F403,F405,
+    # ignore try-except-pass. Bare excepts are caught with B001 and E722
+    S110,
+    # ignore using setattr and getattr with constants
+    B009,B010,
+    # ignore escaping in docstrings: temporary
+    W605,
+    # ignore multiple `##` starting a comment
+    E266,
+
+per-file-ignores =
+    # unused imports, star imports
+    */__init__.py: F401, F403
+    # ignore whitespacing on flags
+    disnake/audit_logs.py:E241
+    # ignore whitespacing on flags
+    disnake/enums.py:E221,E241
+    # ignore whitespacing after `:`
+    disnake/ext/commands/params.py:E241
```

Comment:
These are explicitly where we have `# fmt: off` for multiple lines.

Reply:
I know, but there's no point in only disabling them for these files since we use black for formatting regardless, and don't really care about how flake8 wants to format things, especially with `E221` and `E241`

Reply:
flake8 still catches things that black does not format